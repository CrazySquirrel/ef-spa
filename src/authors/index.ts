export const siastrebov = {
  name: 'Sergei Iastrebov',
  url: 'https://plus.google.com/u/0/102372703586362573921',
};
