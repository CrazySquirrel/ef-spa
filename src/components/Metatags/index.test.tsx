import * as React from 'react';
import {shallow} from 'enzyme';

import {Author, Publisher} from './index';

import {siastrebov} from '../../authors';

describe('Author', () => {
  it('render', () => {
    expect(shallow(<Author {...siastrebov}/>)).toMatchSnapshot();
  });
});

describe('Publisher', () => {
  it('render', () => {
    expect(shallow(<Publisher/>)).toMatchSnapshot();
  });
});
