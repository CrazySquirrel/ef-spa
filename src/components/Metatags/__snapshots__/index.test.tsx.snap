version https://git-lfs.github.com/spec/v1
oid sha256:a00305a5343b502290695ce738602ffb2da00f37276f4070eec357e9176dd7f2
size 13504
